Use netbsd settings for the paths as those match what we want.
Note we patch configure.proto as configure.ac gets generated from that.
diff -up vice-3.4/configure.proto.datadir vice-3.4/configure.proto
--- vice-3.4/configure.proto.datadir	2020-08-22 16:24:14.294644292 +0200
+++ vice-3.4/configure.proto	2020-08-22 16:24:53.355124602 +0200
@@ -3625,7 +3625,7 @@ AC_DEFINE_UNQUOTED(PREFIX, "$PREFIX", [W
 
 dnl Path for keymaps, ROMs etc.
 case "$host_os" in
-  netbsd*)
+  linux*)
     VICEDIR="$PREFIX/share/vice"
     AM_CONDITIONAL(ALTERNATE_DOCDIR, true)
     ;;
@@ -3633,7 +3633,7 @@ case "$host_os" in
     VICEDIR="$PREFIX/lib/vice"
     AM_CONDITIONAL(ALTERNATE_DOCDIR, true)
     ;;
-  linux*)
+  foo*)
     if test x"$host_cpu" = "xx86_64"; then
       VICEDIR="$PREFIX/lib64/vice"
       AM_CONDITIONAL(ALTERNATE_DOCDIR, false)
